frameworks/elastic/universe-kibana/package.json

```diff
@@ -6,7 +6,7 @@
   "name": "kibana",
   "version": "{{package-version}}",
   "maintainer": "support@mesosphere.io",
-  "description": "Kibana 5, and optionally X-Pack",
+  "description": "Kibana 6, and X-Pack",
```

Comment:
Since the `config.json` still has a `kibana.xpack_enabled` option, it would seem that the use of `X-Pack is still optional.

Reply:
in kibana 6 xpack comes inbuilt and by default its enabled